docs/src/osdk/guide/create-project.md

```diff
@@ -13,7 +13,7 @@ Creating a new kernel project is simple.
 You only need to execute the following command:
 
 ```bash
-cargo osdk new --kernel myos
+cargo osdk new --type kernel myos
```

Comment:
Personally, I don't like the `--type` argument. It does not align with Cargo. And it makes the command for a Hello World example longer than necessary. And by the way, `type` is an overloaded word and thus should be avoided in general.

Reply:
There's a short version `-t`.

I'd like to add support for all the usages just as what the profile selection CLI options in Cargo looks like.

```bash
cargo osdk new --type kernel myos
cargo osdk new -t kernel myos
cargo osdk new --kernel myos
cargo osdk new --library kernel mylib
cargo osdk new --lib kernel mylib
cargo osdk new --t lib mylib
cargo osdk new --t library mylib
```
and they are mutually-exclusive.